docs/book/src/topics/rosa/creating-a-cluster.md

```diff
@@ -1,30 +1,83 @@
 # Creating a ROSA cluster
 
 ## Permissions
-CAPA controller requires an API token in order to be able to provision ROSA clusters:
+### Authentication using service account credentials
```

Comment:
ROSA-HCP not ROSA

Reply:
I'm assuming you meant to write that comment one line below.

Not saying I cannot change it, but ...
```
$ git checkout main
$ git grep ROSA docs/book/src/topics/rosa/creating-a-cluster.md
docs/book/src/topics/rosa/creating-a-cluster.md:# Creating a ROSA cluster
docs/book/src/topics/rosa/creating-a-cluster.md:CAPA controller requires service account credentials to be able to provision ROSA clusters:
docs/book/src/topics/rosa/creating-a-cluster.md:1. Create a new kubernetes secret with the service account credentials to be referenced later by `ROSAControlPlane`
docs/book/src/topics/rosa/creating-a-cluster.md:    Note: to consume the secret without the need to reference it from your `ROSAControlPlane`, name your secret as `rosa-creds-secret` and create it in the CAPA manager namespace (usually `capa-system`)
docs/book/src/topics/rosa/creating-a-cluster.md:1. Create a credentials secret within the target namespace with the token to be referenced later by `ROSAControlePlane`
docs/book/src/topics/rosa/creating-a-cluster.md:Follow the guide [here](https://docs.aws.amazon.com/ROSA/latest/userguide/getting-started-hcp.html) up until [Step 3](https://docs.aws.amazon.com/ROSA/latest/userguide/getting-started-hcp.html#getting-started-hcp-step-3) 
docs/book/src/topics/rosa/creating-a-cluster.md:Once Step 3 is done, you will be ready to proceed with creating a ROSA cluster using cluster-api.
docs/book/src/topics/rosa/creating-a-cluster.md:1. Render the cluster manifest using the ROSA cluster template:
docs/book/src/topics/rosa/creating-a-cluster.md:1. If a credentials secret was created earlier, edit `ROSAControlPlane` to reference it:
docs/book/src/topics/rosa/creating-a-cluster.md:    kind: ROSAControlPlane
docs/book/src/topics/rosa/creating-a-cluster.md:    kind: ROSAControlPlane
docs/book/src/topics/rosa/creating-a-cluster.md:see [ROSAControlPlane CRD Reference](https://cluster-api-aws.sigs.k8s.io/crd/#controlplane.cluster.x-k8s.io/v1beta2.ROSAControlPlane) for all possible configurations.
```

Reply:
most of our doc distinguish between ROSA (classic) and ROSA with HCP like the aws doc [here](https://docs.aws.amazon.com/rosa/latest/userguide/getting-started-hcp.html) so better to mention it as "ROSA-HCP" or "ROSA with HCP".

Reply:
Yep. As I said, I can change it. My point was that the .md document in question uses just ROSA. No HCP, no Hypershift.

Reply:
Okay then, lets do it in another PR change all docs at once. 
Would you create an issue mentioning that we need to change ROSA to ROSA-HCP in docs so we don't forget it.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/5257